fedmsg_meta_fedora_infrastructure/pagure.py

```diff
@@ -353,44 +353,40 @@ def subtitle(self, msg, **config):
         elif 'pagure.project.forked' in msg['topic']:
             old_project = msg['msg']['project']['parent']['name']
             tmpl = self._(
-                '{user} forked project "{old_project}" to "{project}"'
+                '{user} forked {old_project} to {project}'
             )
             return tmpl.format(
                 user=user, old_project=old_project, project=project)
         elif 'pagure.pull-request.comment.added' in msg['topic']:
             prid = msg['msg']['pullrequest']['id']
             tmpl = self._(
-                '{user} commented on pull-request#{id} of project '
-                '"{project}"'
+                '{user} commented on PR #{id} on {project}'
             )
             return tmpl.format(user=user, id=prid, project=project)
         elif 'pagure.pull-request.comment.edited' in msg['topic']:
             prid = msg['msg']['pullrequest']['id']
             tmpl = self._(
-                '{user} edited a comment on pull-request#{id} of project '
-                '"{project}"'
+                '{user} edited a comment on PR #{id} on {project}'
```

Comment:
That's a good point. It uses "pull request" everywhere except for the messages which refer to comments, where it says "comment on PR". I just based it on the wording in `github.py` which uses that. I guess it is just to save a little bit of space.

I could update it to "comment on pull request", or we could even use "PR" instead of "pull request" everywhere as that is shorter... in that case I would want to update the Github messages to match. Tell me what you prefer. :-)